prisma/schema.prisma

```diff
@@ -216,17 +219,28 @@ model UserProposalFeedback {
 model AdminInfo {
   id String @id @unique @default(uuid())
 
-  mailReceived String?
-  supervisor String?
-  student String?
-  title String?
-  type String?
-  status String?
+  proposal   Proposal @relation(fields: [proposalId], references: [id], onDelete: Cascade)
+  proposalId String
+
+  mailReceived     String?
+  status           String?
```

Comment:
use an enum with predefined status options (depending on the admin people but we should define a few steps and ask them for feedback)